src/main/docker/bootstrap.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+cd /opt/Artemis
+
+if [ -z "$(ls -A config)" ]; then
+   echo "Config is Empty .. copying default ones .."
+   cp -n -a /defaults/Artemis/. config/
+else
+   echo "Config is not empty .. not copying default configs .."
+fi
+
+# Create sync file that indicates waiting for services
+mkdir -p sync
+touch sync/waiting-backend
+
+# Ensure at least the directories are owned by artemis. "-R" takes too long
+chown artemis:artemis config data sync
```

Comment:
This is very specific to your reverse proxy configuration. I guess the better solution for a "artemis is down" warning would be to introduce a dedicated error page for backend timeout.

Reply:
13-14 I agree . I will delete it :)

Reply:
Resolved. Iff ok, you can mark conversation as resolved :)